sys/sys/tree.h

```diff
@@ -33,6 +33,7 @@
 #define	_SYS_TREE_H_
 
 #include <sys/cdefs.h>
+#include "stdint.h"
```

Comment:
<sys/stdint.h>? I don't think any other headers here use relative includes.

Reply:
which file should be used for boostrapping sys/stdint.h in tools/build? I'm not sure where the ptraddr_t typedef should go.

Reply:
Ah I guess we might need a `tools/build/sys/stdint.h` then :(

Reply:
Or you can use __ptraddr_t